Tasks/git-clone-accesstoken/main.ps1

```diff
@@ -0,0 +1,82 @@
+param (
+    [Parameter()]
+    [string]$RepositoryUrl,
+    [Parameter()]
+    [string]$At,
+    [Parameter()]
+    [string]$Directory,
+    [Parameter()]
+    [string]$Branch
+)
+
+# ====== Create repository folder is it doesnot exist.
+
+try {
```

Comment:
It is worth validating the required parameters before proceeding.

Reply:
Looks like our task validation in service can make sure of required parameters. So it is optional to add the validators here.

Reply:
Thanks for the suggestion. I think we can rely on backend check.